### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 4)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

**Comment by msewaweru** [comment]
Trigger rebuild.

**Review by msewaweru — APPROVED** [review]
LGTM!

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationExtensionProperty.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Retrieves the list of extension properties on an application object.
 
 ### Example 1: Get extension properties
 ```
-PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationExtensionProperty.md:26]: ObjectId GUID not updated in the output.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationExtensionProperty.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Get extension properties'
   code: |-
-    PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84"
+    PS C:\>Get-AzureADMSApplicationExtensionProperty -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb"
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationExtensionProperty.yml:17]: ObjectId GUID not updated in the output.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationOwner.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Retrieves the list of owners for an application object.
 
 ### Example 1: Get the owner of an application
 ```
-PS C:\>Get-AzureADMSApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -Top 1
+PS C:\>Get-AzureADMSApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -Top 1
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationOwner.md:26]: ObjectId GUID not updated in the output.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationOwner.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ This command gets the owner of an application.
 
 ### Example 1: Get the owners of an application
 ```
-PS C:\>Get-AzureADMSApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -All $true
+PS C:\>Get-AzureADMSApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -All $true
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationOwner.md:37]: ObjectId GUID not updated in the output.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationOwner.yml [hunk 5]
```diff
@@ -18,7 +18,7 @@ syntaxes:
 examples:
 - title: 'Example 1: Get the owner of an application'
   code: |-
-    PS C:\>Get-AzureADMSApplicationOwner -ObjectId "3ddd22e7-a150-4bb3-b100-e410dea1cb84" -Top 1
+    PS C:\>Get-AzureADMSApplicationOwner -ObjectId "aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb" -Top 1
```
Comment (msewaweru) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSApplicationOwner.yml:21]: ObjectId GUID not updated in the output.

**Review by msewaweru — CHANGES_REQUESTED** [review]
A few instances where the GUIDs aren't updated in the output data.

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSGroupPermissionGrant.yml [hunk 6]
```diff
@@ -19,7 +19,7 @@ examples:
     		Get-AzureADMSGroupPermissionGrant -Id "4823e767eca44858aed244154009b764" 
 
     		Id             : vsMaSY2k_E7761KhRqpx7OGFvAwvdZnJM1s7Iqkt4PU
-    		ClientId       : deefce9d-be43-4b49-a9d3-851af6d2c26c
+    		ClientId       : 00001111-aaaa-2222-bbbb-3333cccc4444
     		ClientAppId    : ba4e4a78-c352-4e59-b657-81b2b395d32b
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSGroupPermissionGrant.yml:23]: ClientAppId not updated

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.md [hunk 7]
```diff
@@ -43,9 +43,9 @@ This command retrieves a list of all conditional access policies in Azure AD.
 
 ### Example 2: Retrieves a conditional access policy in Azure AD with given Id.
 ```
-PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "6b5e999b-0ba8-4186-a106-e0296c1c4358"
+PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
 
-          Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
+          Id                      : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.md:48]:
```suggestion
          Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.md [hunk 8]
```diff
@@ -43,9 +43,9 @@ This command retrieves a list of all conditional access policies in Azure AD.
 
 ### Example 2: Retrieves a conditional access policy in Azure AD with given Id.
 ```
-PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "6b5e999b-0ba8-4186-a106-e0296c1c4358"
+PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.md:46]:
```suggestion
PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "6b5e999b-0ba8-4186-a106-e0296c1c4358"
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.yml [hunk 9]
```diff
@@ -11,7 +11,7 @@ examples:
   code: |-
     PS C:\> Get-AzureADMSConditionalAccessPolicy
 
-              Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
+              Id                      : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.yml:14]:
```suggestion
              Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.yml [hunk 10]
```diff
@@ -21,9 +21,9 @@ examples:
   summary: ""
 - title: 'Example 2: Retrieves a conditional access policy in Azure AD with given Id.'
   code: |-
-    PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "6b5e999b-0ba8-4186-a106-e0296c1c4358"
+    PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.yml:24]:
```suggestion
    PS C:\> Get-AzureADMSConditionalAccessPolicy -PolicyId "6b5e999b-0ba8-4186-a106-e0296c1c4358"
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.yml [hunk 11]
```diff
@@ -21,9 +21,9 @@ examples:
...
 
-              Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
+              Id                      : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSConditionalAccessPolicy.yml:26]:
```suggestion
              Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.md [hunk 12]
```diff
@@ -49,10 +49,10 @@ This command retrieves a list of all named location policies in Azure AD.
 
 ### Example 2: Retrieves a named location policy in Azure AD with given Id.
 ```
-PS C:\> Get-AzureADMSNamedLocationPolicy -PolicyId 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
+PS C:\> Get-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.md:52]:
```suggestion
PS C:\> Get-AzureADMSNamedLocationPolicy -PolicyId 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.md [hunk 13]
```diff
@@ -49,10 +49,10 @@ This command retrieves a list of all named location policies in Azure AD.
...
 
           OdataType                           : #microsoft.graph.countryNamedLocation
-          Id                                  : 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
+          Id                                  : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.md:55]:
```suggestion
          Id                                  : 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.yml [hunk 14]
```diff
@@ -27,10 +27,10 @@ examples:
   summary: ""
 - title: 'Example 2: Retrieves a named location policy in Azure AD with given Id.'
   code: |-
-    PS C:\> Get-AzureADMSNamedLocationPolicy -PolicyId 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
+    PS C:\> Get-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.yml:30]:
```suggestion
    PS C:\> Get-AzureADMSNamedLocationPolicy -PolicyId 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.yml [hunk 15]
```diff
@@ -27,10 +27,10 @@ examples:
...
 
               OdataType                           : #microsoft.graph.countryNamedLocation
-              Id                                  : 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
+              Id                                  : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSNamedLocationPolicy.yml:33]:
```suggestion
              Id                                  : 1b7f0916-7677-40d8-97a1-d606f4ed8fcf
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSPermissionGrantConditionSet.md [hunk 16]
```diff
@@ -40,7 +40,7 @@ PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSe
 
 ### Example 3: Get a permission grant condition set
 ```
-PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
+PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSPermissionGrantConditionSet.md:43]:
```suggestion
PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
```

File: azureadps-2.0-preview/AzureAD/Get-AzureADMSPermissionGrantConditionSet.yml [hunk 17]
```diff
@@ -32,7 +32,7 @@ examples:
   summary: ""
 - title: 'Example 3: Get a permission grant condition set'
   code: |-
-    PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
+    PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Get-AzureADMSPermissionGrantConditionSet.yml:35]:
```suggestion
    PS C:\>Get-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66"
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru, some changes needed here before merging

**Review by msewaweru — APPROVED** [review]

**Pull request merged**